lib/tasks/users.rake

```diff
@@ -2,12 +2,6 @@ namespace :db do
   task admin: :environment do
     user = AdminUser.find_or_create
```

Comment:
Okay, let's do the following:

1. Add a `puts` to log this action, similar to the other user types
2. Delete the `puts user.api_key` on the following line (seems like a leftover from testing or something), and so you won't need the `user` variable any more either.